docs/content.zh/docs/operations/helm.md

```diff
@@ -76,7 +76,7 @@ The configurable parameters of the Helm chart and which default values as detail
 | rbac.create                                    | Whether to enable RBAC to create for said namespaces.                                                                                                          | true                                                                                                                                                                                                                                                                                           |
 | rbac.nodesRule.create                          | Whether to add RBAC rule to list nodes which is needed for rest-service exposed as NodePort type.                                                              | false                                                                                                                                                                                                                                                                                          |
 | operatorPod.annotations                        | Custom annotations to be added to the operator pod (but not the deployment).                                                                                   |                                                                                                                                                                                                                                                                                                |
-| operatorPod.labels                             | Custom labels to be added to the operator pod (but not the deployment).                                                                                        |                                                                                                                                                                                                                                                                                                |
+| operatorPod.labels                             | Custom labels to be added to the operator pod                                                                                        |                                                                                                                                                                                                                                                                                                |
```

Comment:
the updated descriptions are not correct...
It should say ` Custom labels to be added to the operator pod  and deployment`

Reply:
ok. thanks.